### ISSUE TITLE
Education entries not aligned

### ISSUE THREAD

**Issue by nbecker** [posquit0/Awesome-CV#229]
My CV was fine with an older version of awesome-CV, but I just did a git pull (update to master), and now the alignment is off.
I have 2 education entries.  The 2nd seems to be slightly indented relative to the 1st.  Any ideas on why?
![screenshot_20180830_090651](https://user-images.githubusercontent.com/117456/44853694-99002300-ac34-11e8-8010-4ceec21d3c90.png)

**Comment by Nimita311** [comment]
I think you need to put an explicit blank line after the first entry.

**Comment by nbecker** [comment]
It appears you are correct.
So there needs to be a blank between each \cventry?
Do the examples show this?

**Comment by Nimita311** [comment]
Yeah, the example was formatted in that way but did not tell it explicitly. Without digging into the style sheet, I guess that adding an empty line creates a separate paragraph, otherwise the extra indentation is carried alone.

**Comment by CyberMew** [comment]
I also had to add 
    \vspace*{-4mm}
to remove the vertical spacing between them.

**Comment by nateGeorge** [comment]
@CyberMew where did you put the vspace to remove the extra blank line?  I can't figure out the correct spot to put it.

**Comment by fords** [comment]
How to add dual degree in one university?  This below code is causing an error. 
 \cventry
    {Other Bachelors} % Degree
    {Other Masters} % Degree
    {UC Berkeley} % Institution
    {Berkeley, CA} % Location
    {2011} % Date(s)
